src/main/java/exception/UnknownCommandException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class UnknownCommandException extends Exception {
```

Comment:
Since you are identifying specific exceptional conditions, it helps to add a brief Javadoc documentation to tell what this exception is for.